kratos/mpi/tests/sources/test_data_communicator.cpp

```diff
@@ -730,6 +730,11 @@ KRATOS_TEST_CASE_IN_SUITE(DataCommunicatorSendRecv, KratosMPICoreFastSuite)
     std::vector<double> send_buffer_double = {2.0*world_rank, 2.0*world_rank};
     std::vector<double> recv_buffer_double = {0.0, 0.0};
 
+    std::string send_buffer_string = "test";
+    std::string recv_buffer_string;
+    recv_buffer_string.reserve(4);
```

Comment:
I do not resize the buffers for you (since MPI also doesn't). You need to allocate it explicitly.